### PR TITLE
Small README improvements for NixOS users

### PR DESCRIPTION
**Pull request by tsusanka**
Not for NixOS developers, but for NixOS users.

### CODE REVIEW

File: README.md
Comment (grdddj) [inline README.md:36]: After merging with `incoming`, these port numbers will need to be changed.

BTW, what is the plan with the merge, is it still blocked by the CI Gitlab issues?

File: README.md
Comment (grdddj) [inline README.md:28]: Here in master, we have two nix-shells, so we could specify `in the root directory`  - but not necessary, if we will merge it soon with `incoming`, where we have united the nix-shell

**Review by grdddj — APPROVED** [review]
Green light from me - the comments are just pointing out the merge with `incoming`

**Comment by tsusanka** [comment]
Oh right. Let's merge `incoming` first and I'll rebase this one then.

File: README.md
Comment (tsusanka) [reply README.md:44]: Fixed by rebasing, please check.

File: README.md
Comment (tsusanka) [reply README.md]: Fixed by rebasing, please check.

File: README.md
Comment (grdddj) [reply README.md:44]: Port 9002 is right, and there are no other breaking changes. So we can merge! (When you were the one creating the commit, should it be me to press the Green button?)

Comment (tsusanka) [reply README.md:44]: Yes, please approve the PR and feel free to merge then :).

**Comment by grdddj** [comment]
Approved, LGTM!

**Pull request merged**
